app/models/initial_teacher_training.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: initial_teacher_trainings
+#
+#  id                   :bigint           not null, primary key
+#  programme_end_date   :date
+#  programme_start_date :date
+#  programme_type       :string
+#  result               :string
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#  dqt_record_id        :bigint           not null
+#
+# Indexes
+#
+#  index_initial_teacher_trainings_on_dqt_record_id  (dqt_record_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (dqt_record_id => dqt_records.id)
+#
+class InitialTeacherTraining < ApplicationRecord
```

Comment:
Why are we storing these in the database?

Reply:
I must have gotten the wrong context around associating the record from DQT. Did it mean update the users trn number we got from DQT?